test/e2e/framework/pod/wait.go

```diff
@@ -217,7 +217,7 @@ func WaitForPodsRunningReady(c clientset.Interface, ns string, minPods, allowedN
 				badPods = append(badPods, pod)
 			default:
 				if metav1.GetControllerOf(&pod) == nil {
-					e2elog.Logf("Pod %s is Failed, but it's not controlled by a controller", pod.ObjectMeta.Name)
+					e2elog.Logf("Pod %s INFO: Number of not-ready pods (%d) is below the allowed threshold (%d).", pod.ObjectMeta.Name)
```

Comment:
Huh? This makes the code worse, not better? Is this intentional?

Reply:
I just tried to fix https://github.com/kubernetes/kubernetes/issues/111092,no idea how changing the way a error message is displayed makes the code worse.

Reply:
- The format string doesn't match the parameters.
- The message doesn't match the if check.